Adhesion/Interactions/ES_Coulomb.py

```diff
@@ -0,0 +1,96 @@
+#
+# Copyright 2020 Antoine Sanner
```

Comment:
We will need to rerun the updating of the license files

Reply:
Is this what you meant? I am not sure I understand what I should write in the first line and what the current license file number is.

Reply:
We have an automatic script doing this. 
run `sh maintenance/update_license_headers.sh` from the project main directory once everything else is done with the pull request.